installation/scripts/installer.sh

```diff
@@ -61,7 +57,10 @@ if [ $CR_PATH ]; then
 fi
 
 echo -e "\nApplying installation combo yaml"
-bash ${CURRENT_DIR}/concat-yamls.sh ${INSTALLER} ${INSTALLER_CONFIG} ${CR_PATH} | kubectl apply -f -
+bash ${CURRENT_DIR}/concat-yamls.sh ${INSTALLER} ${INSTALLER_CONFIG} ${CR_PATH} ${DEFAULT_SA_RBAC_ROLE_PATH} | kubectl apply -f -
+
+bash ${CURRENT_DIR}/is-ready.sh kube-system k8s-app kube-dns
```

Comment:
I think we can delete this `k8s-app` as we do not have such NS or deployment/pod